avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/PrepRankingData.scala

```diff
@@ -124,7 +124,7 @@ object PrepRankingData {
     val contentFeaturesWithSpVec = contentFeatures.filter(
       it =>
         CommonUtils.isArrayOfSparseTensor(truncateDf.schema(it).dataType))
-    var transformDf = truncateDf.drop(DefaultRankingId)
+    var transformDf = truncateDf.drop(params.groupIdList: _*)
```

Comment:
Just a second thought, should we keep these ids? they might be useful later in transfer learning